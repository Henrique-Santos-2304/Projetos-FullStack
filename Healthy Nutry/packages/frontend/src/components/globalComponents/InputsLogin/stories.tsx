import { Story, Meta } from "@storybook/react";
import InputsLogin from ".";

export default {
  title: "InputsLogin",
  component: InputsLogin,
} as Meta;

export const Basic: Story = (args) => <InputsLogin {...args} />;

Basic.args = {
  title: "title basic",
  description: "description basic",
};

export const Default: Story = (args) => <InputsLogin {...args} />;
