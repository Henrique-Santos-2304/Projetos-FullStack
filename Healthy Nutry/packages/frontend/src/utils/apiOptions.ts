export const optionsMealTypes = [
  { value: "", label: "Selecione tipo de refeição" },
  { value: "coffee", label: "Café" },
  { value: "lunch", label: "Almoço" },
  { value: "dinner", label: "Janta" },
];

export const optionsProteins = [
  { value: "", label: "Selecione uma Proteína" },
  { value: "fish", label: "Peixe" },
  { value: "ox", label: "Boi" },
  { value: "pig", label: "Porco" },
  { value: "chicken", label: "Frango" },
  { value: "egg", label: "Ovo" },
];

export const optionsCarbohydrates = [
  { value: "", label: "Selecione um Carboidrato" },
  { value: "rice", label: "Arroz" },
  { value: "potato", label: "Batata" },
  { value: "Yam", label: "Inhame" },
  { value: "noodle", label: "Macarrão" },
  { value: "bread", label: "Pão" },
];

export const optionsVegetables = [
  { value: "", label: "Selecione um Vegetal" },
  { value: "zucchini", label: "Abobrinha" },
  { value: "aubergine", label: "Berinjela" },
  { value: "pumpkin", label: "Abóbora" },
  { value: "chicory", label: "Almeirão" },
  { value: "tomato", label: "Tomate" },
];

export const proteinsChoice = ["Peixe", "Boi", "Porco", "Frango", "Ovo"];
export const carbohydrateChoice = [
  "Arroz",
  "Batata",
  "Inhame",
  "Macarrão",
  "Pão",
];
export const vegetableChoice = [
  "Abobrinha",
  "Berinjela",
  "Abóbora",
  "Almeirão",
  "Tomate",
];
