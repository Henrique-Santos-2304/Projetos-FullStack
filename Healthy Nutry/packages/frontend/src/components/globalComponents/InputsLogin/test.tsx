import InputsLogin from ".";
import { screen } from "@testing-library/react";
import { renderTheme } from "utils/testRenderTheme";

describe("<InputsLogin />", () => {
  it("should render ", () => {
    const { container } = renderTheme(<InputsLogin />);
    expect(container).toMatchSnapshot();
    screen.logTestingPlaygroundURL();
  });
});
