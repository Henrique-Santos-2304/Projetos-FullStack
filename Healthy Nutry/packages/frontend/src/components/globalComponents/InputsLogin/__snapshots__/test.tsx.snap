// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<InputsLogin /> should render  1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-align-items: flex-start;
  -webkit-box-align: flex-start;
  -ms-flex-align: flex-start;
  align-items: flex-start;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  padding: 1rem 0.6rem;
  margin: 0.6rem 0;
}

<div>
  <div
    class="c0"
  />
</div>
`;
